backend/app/Jobs/ProcessMqttMessageJob.php

```
@@ -19,82 +20,92 @@ class ProcessMqttMessageJob implements ShouldQueue
     public function __construct(
-        private readonly int    $connectionId,
+        private readonly int $connectionId,
         private readonly string $topic,
         private readonly string $payload,
         private readonly string $receivedAt,
     ) {}
 
-    public function handle(MqttMessageParser $parser, ActionExecutor $executor): void
+    public function handle(MqttMessageParser $parser, ActionExecutor $executor, TenantContext $tenant): void
     {
         $connection = MachineConnection::with(['activeTopics.activeMappings'])->find($this->connectionId);
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\MachineConnection::with().


(staticMethod.notFound)